### PR TITLE
Fix some bugs on frontend

### PR DESCRIPTION
**Pull request by lukicenturi**
Closes #(issue_number)

## Checklist

- [x] Fix chart problem that is not updated when range selector is moved.
- [x] Remove language selector on toolbar
- [x] Fix asset form that is hidden (when adding a new asset)

### CODE REVIEW

File: frontend/app/src/components/dashboard/OverallBalances.vue [hunk 1]
```diff
@@ -173,7 +177,7 @@ const indicator = computed(() => {
   if (get(isLoading)) {
     return '';
   }
-  return get(balanceDelta).isNegative() ? '▼' : '▲';
+  return get(balanceDelta).isNegative() ? 'mdi-menu-down' : 'mdi-menu-up';
```
Comment (kelsos) [inline frontend/app/src/components/dashboard/OverallBalances.vue:180]: why change the unicode character, was there any problem with them?

**Comment by kelsos** [comment]
@lukicenturi can you also provide a list of the things the PR fixes so I can check?

File: frontend/app/src/components/dashboard/OverallBalances.vue (hunk 1 above)
Comment (lukicenturi) [reply frontend/app/src/components/dashboard/OverallBalances.vue:180]: I just think it was too big

**Comment by lukicenturi** [comment]
> @lukicenturi can you also provide a list of the things the PR fixes so I can check?

@kelsos updated

**Review by kelsos — APPROVED** [review]
lgtm @lukicenturi thank you

**Pull request merged**